compose/grafana_conf/provisioning/dashboards/all.yaml

```diff
@@ -0,0 +1,11 @@
+apiVersion: 1
+
+providers:
+- name: 'default'
+  orgId: 1
+  folder: ''
+  type: file
+  disableDeletion: false
+  editable: false
+  options:
+    path: /var/lib/grafana/dashboards
```

Comment:
It should be `dashboard`

Reply:
Using `dashboards` can also work

Reply:
We use `dashboards` here because of the grafana may include multiple Dashboard, not because the word works here 😑